tests/swo/notifications/test_teams.py

```
@@ -11,19 +11,12 @@
 )
 
 
-def test_send_notification_full(mocker, settings):
+def test_send_notification_full(mocker, settings):  # noqa: WPS218
```

Ruff (0.15.18)

[warning] Invalid rule code in `# noqa`: WPS218

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)